mleap-tensorflow/src/main/scala/ml/combust/mleap/tensorflow/TensorflowTransformerOp.scala

```diff
@@ -71,15 +71,18 @@ class TensorflowTransformerOp extends MleapOp[TensorflowTransformer, TensorflowM
       val outputs = outputNames.zip(outputTypes.zip(outputShapes).map {
         case (b, s) => core.types.DataType(b, s).asInstanceOf[TensorType]
       })
-
-      val graph = new org.tensorflow.Graph()
-      val graphDef = GraphDef.parseFrom(graphBytes)
-      graph.importGraphDef(graphDef)
-      TensorflowModel(graph = Some(graph),
+      val modelBytes = format match {
+        case Some("graph") | None => Files.readAllBytes(context.file("graph.pb"))
+        case Some("saved_model") => Files.readAllBytes(context.file("saved_model.zip"))
+        case _ => throw new RuntimeException("Only support `saved_model` and `tf` format");
```

Comment:
should this exception string be `saved_model and graph format`?

and maybe`UnsupportedOperationException` to be a bit more specific? (maybe a few other places too)